quiz.js

```diff
@@ -34,7 +34,26 @@ var dingHandle = setTimeout (writeDing(), 3000);
 var words = ['short', 'medium', 'long', 'interminable', 'lengthy'];
 
 var byLength = function (A, B) {
-  // your code here
+
+
+
+function shortAndLong(short, long) {
+    var words = [];
+        for (var i = 0; i < startNode.answers.length; i++) {
```

Comment:
I appreciate the resourcefulness, but beware of random code from the internet. startNode is not a javascript construct